File: src/main/java/com/example/service/OrderService.java
package com.example.service;

import com.example.dto.order.OrderCreationDto;
import com.example.dto.order.OrderDto;
import com.example.dto.transaction.TransactionDto;

import java.util.List;
import java.util.UUID;

public interface OrderService {
    OrderDto findById(UUID uuid);
    OrderDto save(UUID accountId, UUID orderStatusId, OrderCreationDto creationDto);
    void deleteById(UUID uuid);
}

File: src/main/java/com/example/service/OrderService.kt
package com.example.service

import com.example.dto.order.OrderCreationDto
import com.example.dto.order.OrderDto
import java.util.*

interface OrderService {
    fun findById(id: UUID): OrderDto
    fun addOrder(accountId: UUID, orderCreationDto: OrderCreationDto): OrderDto
    fun changeOrderStatus(orderId: UUID): OrderDto
    fun deleteById(id: UUID)
}